inbc-program/inbc/parser/parser.py

```diff
@@ -85,7 +85,7 @@ def parse_source_args(self) -> None:
 
         # Application Remove Command
         app_remove_parser = app_subparsers.add_parser('remove')
-        app_remove_parser.add_argument('--gpgKeyName', '-gkn', required=True,
+        app_remove_parser.add_argument('--gpgKeyName', '-gkn', required=False,
                                        type=lambda x: validate_string_less_than_n_characters(
                                            x, 'str', 50),
                                        help='GPG key name of the source to remove.')
```

Comment:
**Suggestion:** The 'required' flag for '--gpgKeyName' argument in the 'app_remove_parser' has been changed from True to False. This could lead to unexpected behavior if the code assumes this argument is always provided. Ensure that the application_remove function can handle cases where '--gpgKeyName' is not provided. [possible issue]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
        # It's assumed that the application_remove function is designed to handle an optional GPG key name.
        # If not, the function should be updated accordingly to avoid runtime errors when the argument is not provided.
```

Reply:
Done already.